e2e-tests/functions

```diff
@@ -73,7 +73,6 @@ create_namespace() {
 	local skip_clean_namespace="$2"
 
 	if [[ ${CLEAN_NAMESPACE} == 1 ]] && [[ -z ${skip_clean_namespace} ]]; then
-		delete_crd
 		kubectl_bin get ns \
```

Comment:
if we moved this option to create_infra method should we replace create_namespace to create_infra in tests that use create_namespace method?

Reply:
done